diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -0,0 +1,239 @@
+function StudentQuiz(runtime, element) {
+
+    var studentQuiz = this;
+    studentQuiz.startOver = false;
+    studentQuiz.movingToStep = false;
+
+    /* Javascript for Student view in LMS.*/
+
+    var common = new Common(),
+        form = $(".diagnostic-feedback #student_view_form"),
+        studentViewFormSelector = ".diagnostic-feedback #student_view_form",
+
+        //selectors
+        currentAnswerContainer = ".diagnostic-feedback .current",
+        questionId = '.question-id',
+        selectedStudentChoice = 'input[type="radio"]:checked',
+
+        finalResult = '.diagnostic-feedback #response_body',
+        choiceSelector = '.diagnostic-feedback .answer-choice',
+
+        nextActionSelector = '.diagnostic-feedback ul[role="menu"] a[href*="next"]',
+        previousActionSelector = '.diagnostic-feedback ul[role="menu"] a[href*="previous"]',
+        finishActionSelector = '.diagnostic-feedback ul[role="menu"] a[href*="finish"]',
+        cancelActionSelector = '.diagnostic-feedback ul[role="menu"] a[href*="cancel"]',
+        completedStepSelector = ".diagnostic-feedback #completed_step",
+        studentViewFormSecSelector = ".diagnostic-feedback #student_view_form section",
+        choiceSelectedBtnSelector = "input[type='radio']",
+        exportProgressSelector = '.diagnostic-feedback #export_progress',
+        exportDataBtnSelector = "#export_data";
+
+    function hideActions() {
+        // hide next, previous, finish action button
+        // show start over button
+        $(nextActionSelector + ', '+ previousActionSelector + ', ' + finishActionSelector ).hide();
+        $(cancelActionSelector).show();
+    }
+
+    function resetActions() {
+        // hide start over button
+        // show next, previous, finish action button
+        $(nextActionSelector + ', ' + previousActionSelector).show();
+        $(cancelActionSelector).hide();
+    }
+
+    function showResult(result) {
+        // shows final result of student
+        var imgSrc = result.student_result.img;
+
+        var htmlBody = result.student_result.html_body;
+
+        var html = '<div>';
+        if (imgSrc) {
+            html += '<img class="result-img" src="' + imgSrc + '" alt="No Result image"> ' +
```

Comment:
`alt` text should describe the image (for accessibility), not be used as a placeholder for when the image doesn't load.